hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -166,6 +168,7 @@ public long getSizeInMemory() {
     @Override
     public boolean put(final Object key, final Object value, final long txTimestamp, final Object version) {
         // The calling code has already done the work of checking if any existing cached entry is replaceable.
+        logger.fine(String.format("put(): Key %s, value %s, txTimestamp %d, version %s", key, value, txTimestamp, version));
```

Comment:
LocalRegionCache is the base for all local cache regions. Logging here causes all insertions to be logged. We better not include it here.

Reply:
I'll remove!